doc/Table.md

```diff
@@ -0,0 +1,281 @@
+| dist         | float      | Final distance in phase space
+| distp        | float      | Maximum slope of distance in phase space
+| qx_det       | float      | Horizontal detuning
+| qx_spread    | float      | Spread of horizontal detuning
+| qy_det       | float      | Vertical detuning
+| qy_spread    | float      | Spread of vertical detuning
+| resxfact     | float      | Horizontal factor to nearest resonance
+| resyfact     | float      | Vertical factor to nearest resonance
+| resorder     | int(11)    | Order of nearest resonance
+| smearx       | float      | Horizontal smear
+| smeary       | float      | Vertical smear
+| smeart       | float      | Transverse smear
+| sturns1      | int(11)    | Survived turns 1st particle
+| sturns2      | int(11)    | Survived turns 2nd particle
+| sseed        | float      | Starting seed for random generator
+| qs           | float      | Synchrotron tune
+| sigx2        | float      | Horizontal amplitude 2nd particle
+| sigy2        | float      | Vertical amplitude 2nd particle
+| sigxmin      | float      | Minimum horizontal amplitude
+| sigxavg      | float      | Mean horizontal amplitude
+| sigxmax      | float      | Maximum horizontal amplitude
+| sigymin      | float      | Minimum vertical amplitude
+| sigyavg      | float      | Mean vertical amplitude
+| sigymax      | float      | Maximum vertical amplitude
+| sigxminld    | float      | Minimum horizontal amplitude (linear decoupled)
+| sigxavgld    | float      | Mean horizontal amplitude (linear decoupled)
+| sigxmaxld    | float      | Maximum horizontal amplitude (linear decoupled)
+| sigyminld    | float      | Minimum vertical amplitude (linear decoupled)
+| sigyavgld    | float      | Mean vertical amplitude (linear decoupled)
+| sigymaxld    | float      | Maximum vertical amplitude (linear decoupled)
+| sigxminnld   | float      | Minimumhorizontal amplitude (nonlinear decoupled)
+| sigxavgnld   | float      | Mean horizontal amplitude (nonlinear decoupled)
+| sigxmaxnld   | float      | Maximumhorizontal amplitude (nonlinear decoupled)
+| sigyminnld   | float      | Minimum vertical amplitude (nonlinear decoupled)
+| sigyavgnld   | float      | Mean vertical amplitude (nonlinear decoupled)
+| sigymaxnld   | float      | Maximum vertical amplitude (nonlinear decoupled)
+| emitx        | float      | Emittance Mode I
+| emity        | float      | Emittance Mode II
+| betx2        | float      | Secondary horizontal beta-function
+| bety2        | float      | Secondary vertical beta-function
+| qpx          | float      | Qx
+| qpy          | float      | Qy
+| version      | float      | Dummy1
+| cx           | float      | Dummy2
+| cy           | float      | Dummy3
+| csigma       | float      | Dummy4
+| xp           | float      | Dummy5
+| yp           | float      | Dummy6
+| delta        | float      | Dummy7
+| dnms         | float      | Internal1
+| trttime      | float      | Internal2
```

Comment:
I think we can drop these two columns (I guess these are mainly the columns of `fort.10`, right?)